DataBaseClass.py

```diff
@@ -35,4 +35,8 @@ def poll(self, id, t):
         try:
             return [x for x in filter(lambda x: x[0]>t,self.streams[id])]
         except:
-            return []
\ No newline at end of file
+            return []
+
+    def recordList(self):
```

Comment:
Naming, recordList immediately implies list object, not function.
perhaps getRecords?